setup.cfg

```diff
@@ -47,7 +47,6 @@ install_requires =
     jinja2~=2.10
     numpy~=1.15
     pybind11~=2.4
-    pytest~=5.2
```

Comment:
`pytest` is imported as a dependency in `gt4py.testing` so I don't think it should be removed.

Reply:
Got it. Thanks for looking into it.